Makefile

```diff
@@ -24,13 +23,25 @@ CONTAINER_ENGINE?=docker
 default: all
 all: build-base build-sss
 
+.PHONY: clean
+clean:
+	$(CONTAINER_ENGINE) rmi $(REPO_NAME):test || true
+	$(CONTAINER_ENGINE) rmi $(IMG):$(IMAGETAG) || true
```

Comment:
```suggestion
	$(CONTAINER_ENGINE) rmi $(REPO_NAME):test $(IMG):$(IMAGETAG) || true
```

Reply:
Torn on this, I like using something different simply because it is going to be harder to accidentally push since it's not setup for that.  It isn't something we should be thinking we can push.. thoughts?

Reply:
> Torn on this, I like using something different simply because it is going to be harder to accidentally push since it's not setup for that. It isn't something we should be thinking we can push.. thoughts?

I'm not sure I follow. This is the `clean` target, where we want to remove the exiting images. The two `rmi` can be condensed into a single one. Am I missing something?

Reply:
ack, somehow I missed what the change was.. merging thanks